### PR TITLE
fix(util/emoji): Update emoji-related deps

### PR DESCRIPTION
**Pull request by zharinov**
## Changes:

- Update relevant deps
- Remove unused `@types/node-emoji`
- Fix unsupported emoji test

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [ ] Code inspection only, or
- [x] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: lib/util/emoji.spec.ts [hunk 1]
```diff
@@ -47,7 +47,7 @@ describe('util/emoji', () => {
     });
 
     describe('unsupported characters', () => {
-      const unsupported = '🪆';
+      const unsupported = '🫠';
```
Comment (zharinov) [inline lib/util/emoji.spec.ts:50]: Melting face 🤣

**Comment by rarkins** [comment]
Conflicts!

File: package.json [hunk 2]
```diff
@@ -154,9 +154,9 @@
     "detect-indent": "6.1.0",
     "editorconfig": "0.15.3",
     "email-addresses": "5.0.0",
-    "emoji-regex": "9.2.2",
-    "emojibase": "5.2.0",
-    "emojibase-regex": "5.1.3",
+    "emoji-regex": "1.0.0",
```
Comment (viceice) [inline package.json:157]: Are you sure about the downgrade  from v9 to v1 ?

Comment (zharinov) [reply package.json:157]: Gosh 🤦‍♂️ Thank you

**Review by viceice — APPROVED** [review]

**Comment by viceice** [comment]
Maybe we should group those in future?

**Pull request merged**